File: src/main/java-graphql/io/github/graphql/resolver/ReactableResolver.java
package io.github.graphql.resolver;

import io.github.graphql.model.*;

/**
 * Resolver for Reactable
 */
@javax.annotation.Generated(
    value = "com.kobylynskyi.graphql.codegen.GraphQLCodegen",
    date = "2021-03-06T19:47:48+0800"
)
public interface ReactableResolver {

    @javax.validation.constraints.NotNull
    ReactionConnectionTO reactions(ReactableTO reactableTO, String after, String before, ReactionContentTO content, Integer first, Integer last, ReactionOrderTO orderBy) throws Exception;

}

File: src/main/graphql/io/github/graphql/resolver/ReactableResolver.kt
package io.github.graphql.resolver

import io.github.graphql.model.*

/**
 * Resolver for Reactable
 */
@javax.annotation.Generated(
    value = ["com.kobylynskyi.graphql.codegen.GraphQLCodegen"],
    date = "2021-03-05T19:09:00+0800"
)
interface ReactableResolver {

    @Throws(Exception::class)
    fun reactions(reactableTO: ReactableTO, after: String?, before: String?, content: ReactionContentTO?, first: Int?, last: Int?, orderBy: ReactionOrderTO?): ReactionConnectionTO

}
